crates/pagecache/src/config.rs

```diff
@@ -95,20 +95,10 @@ unsafe impl Send for ConfigBuilder {}
 
 impl Default for ConfigBuilder {
     fn default() -> ConfigBuilder {
-        static SALT_COUNTER: AtomicUsize = ATOMIC_USIZE_INIT;
-
-        #[cfg(unix)]
-        let salt = {
-            let pid = unsafe { libc::getpid() };
-            ((pid as u64) << 32)
-                + SALT_COUNTER.fetch_add(1, Ordering::SeqCst) as u64
-        };
-
-        #[cfg(not(unix))]
```

Comment:
probably better to leave the unix counter as it is. We spin up large numbers of instances concurrently on some testing systems and uptime is not monotonic across processes. Even though the chance of a collision is pretty small, it's better to be correct to totally rule out false negative test results caused by multiple processes operating on the same data concurrently.